packages/node-client/lib/index.ts

```diff
@@ -66,6 +66,7 @@ export interface AdminAxiosProps {
         request?: AxiosInterceptorManager<AxiosRequestConfig>;
         response?: {
             onFulfilled: (value: AxiosResponse) => AxiosResponse | Promise<AxiosResponse>;
+            onRejected?: (error: unknown) => void;
```

Comment:
Is it not returning an `AxiosError | Promise<AxiosError>?`

Reply:
We can. I'm pretty sure Axios only needs it back if you change it, because just returning nothing was working fine, but interceptors in Axios are pretty poorly documented so I'll update it to be consistent with what we're doing on the request. I was avoiding it from the principle of the best code being the code you never write.